obspy/io/css/tests/test_core.py

```diff
@@ -27,6 +28,7 @@ def setUp(self):
         # directory where the test files are located
         self.path = os.path.join(os.path.dirname(__file__), 'data')
         self.filename = os.path.join(self.path, 'test.wfdisc')
```

Comment:
Good call, I've appended the self.filename variables and the test.wfdisc filenames.